### PR TITLE
[Improvement] Use Druid to split multiple sql

### PR DESCRIPTION
**Pull request by ruanwenjun**
## Purpose of the pull request
Right now, we use `;\n` to split the sql and use simple way to remove the comments. This cannot work will if the sql is complex, this PR will use Druid to split sql and remove comments.

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

File: dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-api/src/main/java/org/apache/dolphinscheduler/plugin/datasource/api/datasource/AbstractDataSourceProcessor.java [hunk 1]
```diff
@@ -128,4 +130,9 @@ public boolean checkDataSourceConnectivity(ConnectionParam connectionParam) {
             return false;
         }
     }
+
+    @Override
+    public List<String> splitAndRemoveComment(String sql) {
```
Comment (caishunfeng) [inline dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-api/src/main/java/org/apache/dolphinscheduler/plugin/datasource/api/datasource/AbstractDataSourceProcessor.java:135]: Can you add some new UT for it?

**Review by caishunfeng — DISMISSED** [review]
LGTM

**Comment by caishunfeng** [comment]
Please add some pr description.

File: dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-api/src/main/java/org/apache/dolphinscheduler/plugin/datasource/api/datasource/AbstractDataSourceProcessor.java (hunk 1 above)
Comment (ruanwenjun) [reply dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-api/src/main/java/org/apache/dolphinscheduler/plugin/datasource/api/datasource/AbstractDataSourceProcessor.java:135]: Done. Add some UT case for Oracle, Dorid, Mysql.

**Comment by ruanwenjun** [comment]
> Please add some pr description.

Done

**Review by caishunfeng — DISMISSED** [review]
+1

**Review by caishunfeng — DISMISSED** [review]
+1

**Review by caishunfeng — APPROVED** [review]
LGTM again

**Pull request merged**